### PR TITLE
fix#11 - backend for student landing page

### LINKED ISSUE #11: Backend for student landing page

### PR DESCRIPTION
**Pull request by devansh-299**
Fixes #11 

Changes Done :-
* Added urls/views needed
* Added html files for testing
* Requires login so tested using changes done in #14 but those changes are not included in this PR

### CODE REVIEW

**Comment by devansh-299** [comment]
@ChetanKhanna since the master has been updated, should I update the PR with merge conflicts resolved?

**Comment by ChetanKhanna** [comment]
Yes please do.

File: transfers/views/dashboard_views.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        messages.info(request, 'You have no pending applications')
+
+    if 'ts2ps' in request.POST:
+        print("thesis to practice school")
+    elif 'ps2ts' in request.POST:
+        print("practice school to thesis")
+    return render(request, 'transfers/student_landing.html', {
+        'full_name': full_name,
+        'approved_applications': approved_applications,
+        'pending_applications': pending_applications})
```
Comment (ChetanKhanna) [inline transfers/views/dashboard_views.py]: Turn it into a CBV please.

**Review by ChetanKhanna — CHANGES_REQUESTED** [review]
Thanks for this! I left a comment.

**Review by ChetanKhanna — APPROVED** [review]

**Pull request merged**